test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -495,20 +495,6 @@ test
     await t.expect(resendEmailView.innerText).eql('Haven\'t received a call? Call again');
   });
 
-test
-  .requestHooks(logger, smsPrimaryMock)('Callout appears after 30 seconds at most even after re-render', async t => {
-    const challengePhonePageObject = await setup(t);
-    await challengePhonePageObject.clickNextButton();
```

Comment:
removing this test case as it's not valid anymore - in the SMS challenge flow, clicking the next button results in a new remediation from the server therefore we change views so in that scenario the timestamp SHOULD get cleared and reset.